.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -193,10 +180,10 @@ stages:
             -e GEN2_CAPTURED_SIG_VERSION=${gen2_captured_sig_version} \
             ${CONTAINER_IMAGE} make -f packer.mk convert-sig-to-classic-storage-account-blob
           displayName: Convert Shared Image Gallery To VHD Blob In Classic Storage Account
-          condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), eq(variables['MODE'], 'gen2Mode'))
+          condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), eq(variables['MODE'], 'linuxVhdMode'))
```

Comment:
do we need this is it's our only mode?

Reply:
not necessary but figured I'd keep it for verbosity's sake, though fine with removing if you think that's best